src/services/preProcess.ts

```diff
@@ -13,20 +13,20 @@ namespace ts {
         const importedFiles: FileReference[] = [];
         let ambientExternalModules: { ref: FileReference, depth: number }[];
         let braceNesting = 0;
-        let lastTokenWasDot = false;
+        let lastToken: SyntaxKind = null;

```

Comment:
do not use `null`. `undefined` (or unintialized) is what we have used throughout the code base.

Reply:
Yep, I saw, the linter yelled at me :)